app/commands/generate_missing_mappings_command.rb

```diff
@@ -220,7 +220,9 @@ def generate_and_evaluate_mappings_for_group(unmapped_categories:, embedding_col
       end
     end
 
-    write_disagree_messages(disagree_messages) if disagree_messages.any?
+    write_summary_message(
+      all_generated_mappings, disagree_messages, mapping_data["rules"].size, unmapped_categories[:id_name_map].size
```

Comment:
named params pls given how many we've got?

Reply:
Done.